ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -363,18 +385,188 @@ objects:
         wait "$!"
 
         cp /tmp/artifacts/installer/bootstrap.ign /srv
+        BOOTSTRAP_URI="https://${JOB_NAME_SAFE}-bootstrap-exporter-${NAMESPACE}.svc.ci.openshift.org/bootstrap.ign"
 
-        mkdir -p /tmp/tf
+        # begin bootstrapping
+        if [[ "${CLUSTER_TYPE}" == "aws" ]]; then
+          RHCOS_AMI=ami-0df3f99538fbef10f  # FIXME: assumes AWS_REGION is us-east-1
```

Comment:
you can use something similar to https://github.com/openshift/release/pull/3612

Reply:
> you can use something similar to #3612

Are we committed to that approach?  Seems [like a hack][1] to me.  And if so, this isn't much more of a hack ;).

[1]: https://github.com/openshift/installer/pull/1673#issuecomment-486692546

Reply:
> Are we committed to that approach? Seems like a hack to me. And if so, this isn't much more of a hack ;).
That's only for our testing of UPI. and using the copied rhcos.json atleast makes sure we don't need to bump bootsimages here when we bump it in installer.

Reply:
> That's only for our testing of UPI...

Right, but holding out for a real fix will help motivate us to give customers something so that _they_ don't have to bump bootimages when we bump them in the installer ;).  Dog food, for the win :p

Reply:
> > That's only for our testing of UPI...
> 
> Right, but holding out for a real fix will help motivate us to give customers something so that _they_ don't have to bump bootimages when we bump them in the installer ;). Dog food, for the win :p

but we cannot test the bump in installer if you hard code here.

Reply:
> but we cannot test the bump in installer if you hard code here.

ahh, true.  But we don't have tight doc coupling now, so there's some benefit in testing both the old AMI (which the docs will still be recommending) and the new AMI (which IPI will use).